code/data_structures/src/tree/tree/suffix_array/suffix_array.java

```diff
@@ -0,0 +1,166 @@
+    import java.io.BufferedReader;
+
+    import java.io.IOException;
+
+    import java.io.InputStreamReader;
+
+     
```

Comment:
@Ratna04priya Could you refactored the code and remove unwanted indentation and spacing in code? Also, check for other places in the code.